ResearchKit.xcodeproj/project.pbxproj

```diff
@@ -1944,6 +1953,17 @@
 			name = Misc;
 			sourceTree = "<group>";
 		};
+		CB16A6EB1BB0D0660043CDD8 /* Wait Task */ = {
+			isa = PBXGroup;
```

Comment:
The following still needs to be addressed:
- Move into `Step` folder 
- Refactor folder name to `Wait Step`
- Change from `ORKActiveStep` to `ORKStep`